client/end.py

```diff
@@ -0,0 +1,29 @@
+from definitions import *
+import pygame as pg
+
+from buttonPress import Button
+
+def endScreen(screen, address):
+    print("hello world")
+    button_rejoin = Button(90, 30, "Rejoin", (50,130))
+    backToLogin = Button(200, 30, "Return to Login Screen", (50,180))
+    
+    done = False
+    while not done:
+        for event in pg.event.get():
+            if event.type == pg.QUIT:
+                sys.exit(0)
+            elif event.type == pg.MOUSEBUTTONDOWN:
+                button_rejoin.active = True
+                backToLogin.active = True
+                print("mouse button down")
```

Comment:
please don't leave print statements around; this applies to all changes you're trying to bring in